src/actions/targetActions.js

```diff
@@ -24,4 +24,5 @@ export const deleteTarget = createThunk('DELETE_TARGET', targetService.deleteTar
 
 export const { success: getTargetsSuccess } = getTargets;
 export const { success: createTargetSuccess } = createTarget;
+export const { reset: createTargetReset } = createTarget;
```

Comment:
you can unify in just one sentence: 
`export const { success: createTargetSuccess,  reset: createTargetReset} = createTarget;`

Reply:
Oh, totally right. Thanks!